node-runner-cli/setup/BaseSetup.py

```diff
@@ -59,7 +61,6 @@ def fetch_universe_json(trustenode, extraction_path="."):
 
     @staticmethod
     def generatekey(keyfile_path, keyfile_name, keygen_tag, keystore_password=None, new=False):
```

Comment:
May be we rename this method to `getKeyDetails` as it is no more generating keystore?

Reply:
agreed